src/components/data-exploration/heatmap/HeatmapCellInfo.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Card } from 'antd';
+import PropTypes from 'prop-types';
+
+const HeatmapCellInfo = (props) => {
+  const {
+    cellId, geneName, geneExpression, coordinates,
+  } = props;
+
+  const cellInfoStyle = { fontSize: '0.75rem' };
+
+  const renderCellInfo = () => (
+    <Card
+      size='small'
+      style={{
+        zIndex: 6,
+        border: 0,
+        position: 'absolute',
+        left: `${coordinates.x + 20}px`,
+        top: `${coordinates.y + 20}px`,
+        pointerEvents: 'none',
+      }}
+    >
```

Comment:
Perhaps abstract this styling to a common component between `HeatmapTracksCellInfo`, `CellInfo` (rename to `EmbeddingCellInfo`), `HeatmapCellInfo`